components/console-backend-service/pkg/tracing/tracer.go

```diff
@@ -0,0 +1,105 @@
+package tracing
```

Comment:
Why is it copied from https://github.com/99designs/gqlgen-contrib/blob/master/gqlopentracing/tracer.go? Simply add it to dep.

```toml

[[override]]
  name = "github.com/99designs/gqlgen"
  version = "v0.7.2"

[[constraint]]
  name = "github.com/99designs/gqlgen-contrib"
  revision = "c654377d611cfcf5b77012b5814af5a14a79bc86"
```

Reply:
As agreed offline:
- we will leave the file since it contains modifications
- we will add comment with reference to original file